libs/designer/src/lib/core/index.ts

```diff
@@ -35,7 +35,7 @@ export {
 export { useIsDesignerDirty, resetDesignerDirtyState } from './state/global';
 export { useAllSettingsValidationErrors } from './state/setting/settingSelector';
 export { useAllConnectionErrors } from './state/operation/operationSelector';
-export { serializeWorkflow } from './actions/bjsworkflow/serializer';
+export { serializeWorkflow, parseWorkflowParameterValue } from './actions/bjsworkflow/serializer';
 export {
```

Comment:
There should not be a need to parse the parameter value in teh portal... our template designer should always give out correct values according to type for the parameters read during create.